### PR TITLE
Update manual patched assembly code for OpenSSL 3.0 Windows

### PR DESCRIPTION
**Pull request by jinghe-INTC**
Except these files, for non-LVI, CF, some for LOAD it will use the files generated by Perl scripts with the customized template.

### CODE REVIEW

File: Windows/build_package.cmd [hunk 1]
```diff
@@ -112,17 +112,12 @@ goto end_copying
...
-copy /y  windows\x86_64-mont5.asm %OPENSSL_VERSION%\crypto\bn
+xcopy /Y /Q /S  Windows\crypto %OPENSSL_VERSION%\crypto
+goto end_copying
 
 :extra_copying_cf
 
-copy /y  windows\aesni-x86_64.asm %OPENSSL_VERSION%\crypto\aes
-copy /y  windows\vpaes-x86_64.asm %OPENSSL_VERSION%\crypto\aes
-copy /y  windows\x86_64cpuid.asm %OPENSSL_VERSION%\crypto
+rem empty for now
```
Comment (markg-intel) [inline Windows/build_package.cmd:120]: is the idea that changing the perl file will take care of the RET instructions?

File: Windows/build_package.cmd
Comment (markg-intel) [inline Windows/build_package.cmd:1]: maybe we should change this to powershell (.ps1). it is 2023 :)

**Review by markg-intel — APPROVED** [review]
i didn't really look at the .asm Windows assembly files. what i've said is that you only have to analyze either linux or windows, not both, and use the one you do as a guide for the other. i hlpe this is true. :)

one question: in linux we saw the addition of a "CET epilog" to several assembly files, anything like this in Windows?

**Comment by jinghe-INTC** [comment]
> i didn't really look at the .asm Windows assembly files. what i've said is that you only have to analyze either linux or windows, not both, and use the one you do as a guide for the other. i hlpe this is true. :)
> 
> one question: in linux we saw the addition of a "CET epilog" to several assembly files, anything like this in Windows?

`CET epilog` is only applicable for Linux in the recent OpenSSL 3.0 releases.

File: Windows/build_package.cmd
Comment (jinghe-INTC) [reply Windows/build_package.cmd:1]: I will investigate on it. 
One thing is that it needs to set the proper execution policy to run a PS1 script, and we should not break the build of  the DCAP/QvE project in Windows.

**Review by ShundaZhang — APPROVED** [review]

**Pull request merged**